server/src/App/Models/BillingCompany.php

```diff
@@ -0,0 +1,423 @@
+<?php
+declare(strict_types=1);
+
+namespace Loxya\Models;
+
+use Adbar\Dot as DotArray;
+use Carbon\CarbonImmutable;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Support\Str;
+use Loxya\Config\Config;
+use Loxya\Contracts\Serializable;
+use Loxya\Models\Traits\Serializer;
+use Loxya\Support\Validation\Validator as V;
+use Psr\Http\Message\UploadedFileInterface;
+
+/**
+ * Société de facturation
+ *
+ * @property-read ?int $id
+ * @property string $name
+ * @property UploadedFileInterface|string|null $logo
+ * @property-read string|null $logo_real_path
+ * @property string $street
+ * @property string $postal_code
+ * @property string $locality
+ * @property int $country_id
+ * @property string|null $phone
+ * @property string|null $email
+ * @property string|null $website
+ * @property string|null $vat_number
+ * @property string|null $code1_label
+ * @property string|null $code1_value
+ * @property string|null $code2_label
+ * @property string|null $code2_value
+ * @property-read Country $country
+ * @property-read CarbonImmutable $created_at
+ * @property-read CarbonImmutable|null $updated_at
+ * @property-read CarbonImmutable|null $deleted_at
+ *
+ * @method static Builder|static search(string|string[] $term)
+ */
+final class BillingCompany extends BaseModel implements Serializable
+{
+    use Serializer;
+    use SoftDeletes;
+
+    private const string LOGO_BASEPATH = (
```

Comment:
Ce typage est donc à revert, voir [ce commentaire](https://github.com/Loxya/Loxya/pull/420#issuecomment-3310217156) pour plus d'informations.

```suggestion
    private const LOGO_BASEPATH = (
```